kernel/posix/pthread_sched.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <posix_sched.h>
+
+static bool valid_posix_policy(int policy)
+{
+	if ((policy != SCHED_FIFO) && (policy != SCHED_RR)) {
```

Comment:
Nitpick: please remove the excessive parenthesis.

Reply:
I feel removing internal braces (if (policy != SCHED_FIFO && policy != SCHED_RR) ) will lose the readability. So keeping the braces is better. What is your thought ?

Reply:
I don't have strong opinions on this -- I find it less visually polluted without the parenthesis, and very readable.  It's just a nitpick, we can agree to disagree.